### ISSUE TITLE
Add development bundles to ignored bundles for Healthcheck

### ISSUE THREAD

**Issue by henrykuijpers** [shinesolutions/puppet-aem-curator#208]
At least the following bundles should be there, according to the Adobe Security Checklist:
```
        value            => [
          'org.apache.sling.jcr.webdav',
          'org.apache.sling.jcr.davex',
          'com.adobe.granite.crx-explorer',
          'com.adobe.granite.crxde-lite',
        ],
```

https://github.com/shinesolutions/puppet-aem-curator/blob/8bff6a52519c6f8216c60ee639dee2fa31d46118/manifests/config_aem.pp#L75

The Twitter one shouldn't be needed by the way, you can use the ACS Commons package with the `min`-classifier. :)

**Comment by mbloch1986** [comment]
@henrykuijpers Thanks ! Do you have a link to the Adobe Security checklist where they are recommending it ?

Thanks !

**Comment by cliffano** [comment]
I think it's https://experienceleague.adobe.com/docs/experience-manager-65/administering/security/security-checklist.html?lang=en#security .

```
Check if the CRX development bundles are present
These development OSGi bundles should be uninstalled on both author and publish productive systems before making them accessible.

Adobe CRXDE Support (com.adobe.granite.crxde-support)
Adobe Granite CRX Explorer (com.adobe.granite.crx-explorer)
Adobe Granite CRXDE Lite (com.adobe.granite.crxde-lite)
```

So if the above bundles are not installed as per security guideline, then the fact that those bundles don't exist shouldn't impact the health check.

**Comment by cliffano** [comment]
@henrykuijpers @Carnifrex thanks very much for the PR. our apologies that it takes a year.

this will be included in 3.2.0 .

**Comment by cliffano** [comment]
@henrykuijpers @Carnifrex thanks very much for the PR. our apologies that it takes a year.

this will be included in 3.2.0 .
